x-pack/plugins/fleet/public/components/package_policy_actions_menu.tsx

```diff
@@ -24,13 +24,21 @@ export const PackagePolicyActionsMenu: React.FunctionComponent<{
   packagePolicy: InMemoryPackagePolicy;
   viewDataStep?: EuiStepProps;
   showAddAgent?: boolean;
+  isOpen?: boolean;
   upgradePackagePolicyHref: string;
-}> = ({ agentPolicy, packagePolicy, viewDataStep, showAddAgent, upgradePackagePolicyHref }) => {
+}> = ({
+  agentPolicy,
+  packagePolicy,
+  viewDataStep,
+  showAddAgent,
+  upgradePackagePolicyHref,
+  isOpen = false,
```

Comment:
I have added this isOpen prop to aid testing, I could've fired a click event but I didn't think it was unreasonable for the component to have this prop?

Reply:
@hop-dev nitpick what do you think of naming this `defaultIsOpen` as the way I saw it it's the default value for the state correct?